README.md

```diff
@@ -233,32 +250,35 @@ $ tree my-component
 First, install the cli and then use `manifest-gen` to do this.
 
 ``` shell
-$ cd modality-probe-cli
+$ cd ../modality-probe-cli
 $ cargo install --path .
-$ cd ../
+$ cd ../modality-probe-capi
 $ modality-probe manifest-gen \
-    --lang rust \
-    --file-extension rs \
+    --lang c \
+    --file-extension c \
     --component-name example-component \
     --output-path example-component \
-    examples/rust-example
+    examples/c-example
 ```
 
 Next, we'll want to generate the source code that gives those symbols
 we discussed in the code snippet examples in the previous section
 their definitions. To do that, we'll use `header-gen`:
 
 ```shell
+$ mkdir -p ../examples/c-example/include
```

Comment:
Same as above, we should probably write these commands from the root of the repo, so not prefaced with `..`

Reply:
:+1: Inherited from original `modality-probe-capi` README.md. Changing to:

`$ mkdir -p /examples/c-example/include`